receiver/redisreceiver/metadata.yaml

```diff
@@ -344,6 +344,16 @@ metrics:
     gauge:
       value_type: int
     attributes: [db]
+
+  redis.cluster.slots_assigned:
+    enabled: true
```

Comment:
please mark as disabled by default. You can turn that on by default in a subsequent PR as a breaking change.

Reply:
Addressed.